enhancements/testdata-separation.md

```diff
@@ -0,0 +1,109 @@
+---
+title: testdata-separation
+authors:
+  - "@theishshah"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-07-12
+last-updated: 2021-07-20
+status: TBD
+---
+
+# testdata-separation
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+The current bundle includes both an operator as well as its testdata. This results in inflated binaries and scenarios where users may want to release an operator bundle sans testdata but are unable to do so. The solution to this issue is to provide users the ability to use and generate a slim bundle which is decoupled from the testing necessities. Due to the nature of the testdata, it needs the rest of the bundle so it has limited value alone.
+
+## Motivation
+
+The motivation for this EP is to provide users with the option to deliver a slim bundle without testdata.
+* Creates a smaller size bundle to ship
+* Prevents including testing resources in shipped bundle
+  * Stop potential elevated RBAC's in the test bundle from being included in production environments
+
+### Goals
+
+The goal is to create a solution which allows developers to continue to access and utilize the testdata while still having the option to ship an operator without said testdata in the bundle.
+
+### Non-Goals
+
+The goal of this EP is not to change the contents of the bundle nor the testdata, but just to reorganize them, either with multiple bundles or by allowing the tools to modify a bundle to achieve the desired state.
+
+## Proposal
+
+Please note that this proposed solution was elevated from the Alternatives selection after discussions, and the previous implementation has been moved down.
+
+Hard Split
+* Split existing bundle in a testdata only bundle paired with a operator essentials only bundle
+  * Allows the bundle to save significant space/reduce its size
+  * Allows operator developers to securely ship with including potentially sensitive testdata
+    * Users can use elevated permissions in testing without concerns about shipping those configs
+* Primary impact lies within scorecard internals, as well as scaffolding
+  * Scorecard needs logic to understand whether or not there are two bundle or one
+    * Recombine bundles at scorecard runtime
+      * Scorecard runs continue to behave the same, all this forces is a pre-processing step
+    * Allows scorecard test developers to contiue to use `/bundle` context
+* Allow discovery of test bundle using Annotation path
+  * Scorecard currently assumes testdata in the in current bundle, but can read the annotation to find another bundle
```

Comment:
What happens if the testdata bundle referenced by the operator bundle does not exist?

If multiple references are allowed, what happens when some references exist and others don't?